test/Lachain.CoreTest/RPC/HTTP/Web3/AccountServiceWeb3Test.cs

```diff
@@ -115,6 +117,19 @@ public void Test_GetBalance()
             Assert.AreEqual(balNew, "0x115557b419c5c1f3fa018400000000");
         }
 
+        [Test]
+        // Changed GetBalance to public
+        public void Test_GetBalance_pending()
+        {
+            var address = "0x2605c1ad496f428ab2b700edd257f0a378f83750";
+
+            Execute_dummy_transaction(false);
```

Comment:
These types of function names are wrriten in BlockCase. 
Let's change it to 
"ExecuteDummyTransaction"